### ISSUE TITLE
What can i select echart type to achieve it?

### ISSUE THREAD

**Issue by shiwanjun1995** [apache/echarts#19154]
### Version

5.2.2

### Link to Minimal Reproduction

https://codesandbox.io/s/swny4r?file=/index.js

### Steps to Reproduce

none

### Current Behavior

none

### Expected Behavior

![20230921-143947](https://github.com/apache/echarts/assets/38515714/e7dfc99a-e10c-4656-b621-e725bfda37b8)

### Environment

```markdown
- OS: macOS Monterey
- Browser: Chrome 96.0.4664.55
- Framework: Vue@3
```

### Any additional comments?

I am eager to hear good news from you! @o

**Comment by shiwanjun1995** [comment]
@Ovilia

**Comment by shiwanjun1995** [comment]
It seems like https://github.com/apache/echarts/issues/7316.

**Comment by helgasoft** [comment]
see [nested pies example](https://echarts.apache.org/examples/en/editor.html?c=pie-nest)

**Comment by shiwanjun1995** [comment]
Yes, but the ratio of total must be below 100%.
What i describe will be above 100%.

**Comment by Ovilia** [comment]
You can use pie series or sunburst. Don't worry about the 100% because you can always use label.formatter to control what's displayed.

**Comment by shiwanjun1995** [comment]
Yeah. I don't worry and i did with your suggest to view it likes below.
![截屏2023-09-26 下午5 24 23](https://github.com/apache/echarts/assets/38515714/7447b977-0170-45ce-9477-97fc3238b4fb)
And what i want will to be like that:(another)
<img width="1394" alt="截屏2023-09-26 下午5 26 29的副本" src="https://github.com/apache/echarts/assets/38515714/493f61b0-7c66-425e-a9e1-ce2fe66133e1">

**Comment by Ovilia** [comment]
I think the dark gray color in your design is misleading because it looks like the value is smaller than 100%. Bar series in the polar system uses `roundCap` to make sure the value over the max value is understandable. Check [this demo](https://echarts.apache.org/examples/zh/editor.html?c=polar-roundCap) to find out if it is feasible to your situation.

**Comment by shiwanjun1995** [comment]
I select polar and try my best to achieve it will be, my god!
 
<img width="713" alt="截屏2023-09-28 上午11 34 03" src="https://github.com/apache/echarts/assets/38515714/1b892214-7098-48d9-938e-54c5adcdd5e4">

It cannot describe the total with request or limit relation.

And the picture come from aliyun.
